style.css

```diff
@@ -1,8 +1,20 @@
 h1 {
   font-family: helvetica;
-  color: blue;
+  color: black;
 }
 
 html {
   background-color: green;
+  font-family: sans-serif;
+}
```

Comment:
html should be the first listed. Think of it as a global attribute since it's being applied to all of the html.